### ISSUE TITLE
The structure map is  the ground truth?

### ISSUE THREAD

**Issue by BolinDeng98** [lianqi1008/MCM#6]
Hello. It appears that you are directly using the ground truth of the structure map for Structure Complexity Modeling. Is that correct?

**Comment by lianqi1008** [comment]
Yes, for the dataset in the paper, I directly used segmentation ground truth annotations.
